.github/workflows/action-send-ci-cd-status-slack-message.yml

```
@@ -0,0 +1,99 @@
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+      infra_status:
+        required: true
+        type: string
+        description: "Status of the infrastructure deployment job"
+        default: "skipped"
```

actionlint

input "infra_status" of workflow_call event has the default value "skipped", but it is also required. if an input is marked as required, its default value will never be used

(events)